client-admin/src/actions/index.js

```diff
@@ -559,7 +559,6 @@ const onFbLoginOk = (response, dest, dispatch, optionalPassword) => {
     processFacebookFriendsData(response, dest, dispatch, optionalPassword),
     err => {
       console.error(err)
-      console.dir(arguments)
```

Comment:
Linter say that `arguments` doesn't work in double arrow functions anyhow

Reply:
it's possible that `...arguments` works

Reply:
Actually, i think i misunderstood, so this could prob come back. I thought `arguments` would be unavailable, but I now understand that it'll just spit out the arguments of $.when(...), which is helpful

Reply:
Putting this back in and marking as resolved :)

Reply:
bah. nevermind. linting throws error since arguments not available here. leaving it out

Reply:
```
/Users/patcon/repos/polisCode/client-admin/src/actions/index.js
  564:19  error  'arguments' is not defined  no-undef
```